src/libraries/System.Diagnostics.PerformanceCounter/tests/MutexTests.cs

```diff
@@ -0,0 +1,49 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Security.AccessControl;
+using System.Security.Principal;
+using System.Threading;
+using Xunit;
+
+namespace System.Diagnostics.Tests
+{
+    public static class MutexTests
+    {
+        [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsNotInAppContainer))] // Can't create global objects in appcontainer
+        public static void VerifySecurityIdentifier()
+        {
+            string mutexName = $"{Guid.NewGuid():N}";
+
+            Mutex mutex = null;
+
+            // We can't test with the same global mutex used by performance counters, since the mutex was likely already
+            // initialized elsewhere and perhaps with with different ACLs, so we use a Guid to create a new mutex and
+            // then simulate the behavior by calling into EnterMutex() like the performance monitor code does.
+#pragma warning disable CS0436 // Type conflicts with imported type
+            NetFrameworkUtils.EnterMutex(mutexName, ref mutex);
```

Comment:
So this is just testing the utility code without invoking the product at all?  Could we instead do something to ensure the product code creates the Mutex, then open the Mutex created by the product and examine the ACL?  I think the test should be able to access it since it's running as the same user.